examples/txTransfer.ts

```diff
@@ -0,0 +1,54 @@
+import { UserFactory } from "../libs/user";
+import { config } from "./appConfig";
+
+const main = async () => {
+  let userSender;
+  let userRecipient;
+  try {
+    // # 1 Create an instance of User
+    userSender = await UserFactory.create({
+      blockchainWsUrl: config.blockchainWsUrl,
+      clientApiUrl: config.clientApiUrl,
+      ethereumPrivateKey: config.ethereumPrivateKey,
+      nightfallMnemonic: config.nightfallMnemonic,
+    });
+
+    // # 2 [OPTIONAL] For this example, we create a 2nd instance
+    userRecipient = await UserFactory.create({
+      blockchainWsUrl: config.blockchainWsUrl,
+      clientApiUrl: config.clientApiUrl,
+      ethereumPrivateKey: config.ethereumPrivateKey,
+    });
+
+    // # 3 Make transfer
+    const tokenAddress = config.tokenAddress;
+    const tokenStandard = "ERC20";
+    const value = "0.0001";
+    const txReceipts = await userSender.makeTransfer({
+      tokenAddress,
+      tokenStandard,
+      value,
+      recipientAddress: userRecipient.zkpKeys.compressedZkpPublicKey,
+    });
+    console.log("Transaction receipts", txReceipts);
+
+    // # 4 [OPTIONAL] You can check the transaction hash
+    // COMING SOON
+
+    // # 5 [OPTIONAL] You can check transfers that are not yet in a block
+    const pendingTransfers = await userSender.checkLayer2PendingSpentBalances(
```

Comment:
Do you think that `checkLayer2PendingSpentBalances` is a good name? maybe it is, I'm just wondering if you could enhance it.

Reply:
As of PR #75 we will use `checkPendingTransfers` !! At least more consistent with the other methods